src/sentry/static/sentry/app/components/group/inboxBadges/eventAnnotation.tsx

```diff
@@ -9,13 +9,22 @@ import Tag from 'app/components/tag';
  */
 
 type Props = {
-  annotation: string;
+  annotation?: string;
+  htmlAnnotation?: string;
   to?: {
     pathname: string;
     query: Query;
   };
 };
 
-const EventAnnotation = ({annotation, to}: Props) => <Tag to={to}>{annotation}</Tag>;
+const EventAnnotation = ({annotation, htmlAnnotation, to}: Props) => (
+  <Tag to={to}>
+    {htmlAnnotation ? (
+      <span dangerouslySetInnerHTML={{__html: htmlAnnotation}} />
```

Comment:
Where does `htmlAnnotation` come from? Is it user-generated? Is it sanitized?

Reply:
It comes from the BE, generated by BE not user, so this is safe

Reply:
@billyvg from integrations, this is how the issues stream displays jira issues currently